backbone.layoutmanager.js

```diff
@@ -262,11 +262,6 @@ var LayoutManager = Backbone.View.extend({
     return this.__manager__.renderDeferred.promise();
   },
 
-  // Proxy `then` for easier invocation.
-  then: function() {
-    return this.promise().then.apply(this, arguments);
```

Comment:
How to do that though? I tried doing 
```js
then: function() {		
  return this.promise().then.apply(null, arguments);
}
```
Didn't work

Did not try

```js
then: function() {		
  return this.promise().then();
}
```
though, not sure if it matters